src/bindgen/parser.rs

```diff
@@ -541,7 +541,7 @@ impl Parse {
                     );
                     return;
                 }
-                let path = Path::new(function.sig.ident.to_string());
+                let path = Path::new(escape_raw_identifier(function.sig.ident.to_string()));
```

Comment:
Maybe this should be done in `Path::new`? Maybe somewhere else?

This is an issue for fields and everything though...

Reply:
I believe we can put them into `Function::load` in this context at least.
Would you mind telling me more about `Path`? It's not very clear to me what it represent :)

Reply:
Hmm, that's still not the right thing I fear. We kinda need it for all the identifiers we read out from syn...

`Path` is basically a name and a set of generics that represent a struct. I think the idea why it's a `Path` is in case we eventually supported modules or such (so `foo::bar::Baz` would be a path for example). I think right now they're only identifiers.

Reply:
> ... I think right now they're only identifiers.

In that case, putting raw identifier stripping into `Path::new` sounds reasonable. I'll write more tests case since now this PR affect more than function identifier.

Reply:
Right, Path are identifiers at the moment, but there's a lot of identifiers that aren't paths (such as field names).

I think we should try to do it at the syn level. Maye add an `to_unscaped_string` to `syn::Identifier` in an extension trait, and use it across the codebase?

Reply:
Just found that `syn` has already done that for us, [syn::ext::IdentExt][link].

I'll check usages of `syn::Ident::to_string` and see what's the better strategy to apply `unwraw`to them.

[link]: https://docs.rs/syn/1.0.16/syn/ext/trait.IdentExt.html#tymethod.unraw

Reply:
Yeah, using unraw sounds great!

Reply:
What's your opinion about creating a new constructor for `Path` which like `Path::new_from_ident(&syn::Ident) -> Path`? It could make the (current) relation between `Path` and `syn::Ident` more clear, also reduce the change which forgetting to use `unraw`.

Reply:
Ah, sorry for missing this. That looks ok though probably it's not needed. Just calling `unraw()` at the right places should do.

Reply:
ok，let me modify them